spyder/widgets/about.py

```diff
@@ -105,70 +161,128 @@ def __init__(self, parent):
             the <a href="http://www.famfamfam.com/lab/icons/silk/">FamFamFam
             Silk icon set</a> 1.3 (&copy; 2006 Mark James; CC-BY 2.5), and
             the <a href="https://www.kde.org/">KDE Oxygen icons</a>
-            (&copy; 2007 KDE Artists; LGPL 3.0+).</small>
+            (&copy; 2007 KDE Artists; LGPL 3.0+).
             </p>
```

Comment:
Would you mind adding the credits for the splash screen photo (referenced in #15956) in this PR? I think this is the best place for it, but please feel free to add it somewhere else in `Legal`.

```suggestion
            </p>
            <p>
            Splash screen photo by 
            <a href="https://unsplash.com/@benchaccounting?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Bench Accounting</a> on <a href="https://unsplash.com/?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a>
            </p>
```